docs/3_guides/2.2_modify_zkquiz_questions.md

```diff
@@ -0,0 +1,62 @@
+# Modify ZkQuiz Questions
+
+In [Build your first Aligned Application](2_build_your_first_aligned_application.md), we show how to build a trivia application, called ZkQuiz. ZKQuiz asks the user three questions, and if answered correctly, generates a ZK Proof of the correct answers, posts the proof on Aligned, and upon verification, mints an NFT via a smart contract.
+
+In this guide, we will show you how to replace those questions with others.
+
+{% hint style="warning" %}
+This guide assumes you have already read [Build your first Aligned Application](2_build_your_first_aligned_application.md)
+{% endhint %}
+
+## 1. Modify the Questions Asked
+
+First, we need to modify the questions presented to the user. To do this, navigate to our example in `examples/zkquiz/quiz/script/src/main.rs` and change the questions as needed:
+
+```Rust
+let mut user_answers = "".to_string();
+
+let question1 = "What is the capital of France?";
+let answers1 = ["Berlin", "Madrid", "Paris"];
+user_answers.push(ask_question(question1, &answers1));
+
+let question2 = "What is the chemical symbol for gold?";
+let answers2 = ["Au", "Ag", "Fe"];
+user_answers.push(ask_question(question2, &answers2));
+
+let question3 = "What is the native cryptocurrency of Ethereum?";
+let answers3 = ["Bitcoin", "Ether", "Litecoin"];
+user_answers.push(ask_question(question3, &answers3));
+```
+
+## 2. Update the Program
+
+Now we need to update the program to be proven with the new correct answers. As we saw in [Build your first Aligned Application](2_build_your_first_aligned_application.md), the program in `examples/zkquiz/quiz/program/src/main.rs` takes the user answers as inputs and checks that the hash of these inputs matches the expected output. Therefore, we need to update the expected output with the hash of our new correct answers. The updated example looks as follows:
```

Comment:
For this section, I suggest including a brief example of how you obtained the expected bytes for the output. It doesn't need to be working code to calculate the sha3 hash, but maybe just mention that this output is the result of hashing sha3("cab"), as those are the correct answers in this case.